_author_instructions/05_tutorials.md

```diff
@@ -7,8 +7,11 @@ excerpt: What is a publishable tutorial?
 permalink: /authors/tutorials/
 ---
 
-## What is a publishable tutorial?
-Tutorials are articles that take users through specific tasks or sequences of tasks to help the users learn how to do these tasks on their own for other applications or purposes. 
+## What is a publishable tutorial and/or training article?
```

Comment:
Probably should define the difference after presenting the two archetypical types?